Feature/rdk 56050

dp0000 opened the pull request.

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: Migration to dev_sprint_25_2 from 117308 (patchset 12)
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: Migration to dev_sprint_25_2 from 117308 (patchset 12)
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

dp0000 committed:
Delete AampConfig.cpp.rej

removed .rej

dp0000 committed:
Delete aampgstplayer.cpp.rej

dp0000 committed:
Delete CMakeLists.txt.rej

Deepikasri1234 committed:
Merge branch 'feature/RDK-56050' of https://github.com/rdkcentral/aamp into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=128) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=128) before we can accept your contribution.<br/>**21** out of **25** committers have signed the CLA.<br/><br/>:white_check_mark: shripadbpersonal<br/>:white_check_mark: dp0000<br/>:white_check_mark: pstroffolino<br/>:white_check_mark: nu641001<br/>:white_check_mark: rajat698<br/>:white_check_mark: srikanthreddybijjam-comcast<br/>:white_check_mark: jfagunde<br/>:white_check_mark: molakalapalliharipriya<br/>:white_check_mark: shripadgithub<br/>:white_check_mark: Vinish100<br/>:white_check_mark: anjali-syna<br/>:white_check_mark: rekhap2kandhavelan<br/>:white_check_mark: DomSyna<br/>:white_check_mark: anshephe<br/>:white_check_mark: Gnanesha<br/>:white_check_mark: ALSAMEEMA<br/>:white_check_mark: jameslofthouse-RED<br/>:white_check_mark: varatharajan568<br/>:white_check_mark: psiva01<br/>:white_check_mark: nebutler963<br/>:white_check_mark: Abhi-jith-S<br/>:x: Deepikasri1234<br/>:x: asasik397<br/>:x: VNagori<br/>:x: rkandh015<br/><hr/>**asasik397** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=128) it.</sub>

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
Merge remote-tracking branch 'origin/dev_sprint_25_2' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
Merge branch 'develop' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

molakalapalliharipriya committed:
RDKEMW-5467 : Set PlayerState to IDLE during stop

Reason for change: EPG currently behaves badly when state is set to eSTATE_RELEASED instead of eSTATE_IDLE after stopping
Test Procedure: Tune to any linear channels, Try changing the channels and see tune was successful with out any blue screen
Risks: Medium

Signed-off-by: haripriya_molakalapalli <haripriya_molakalapalli@comcast.com>

Abhi-jith-S committed:
VPLAY-9946: Playback failed in for certain live channels

Reason for change: Clear duration value in mdhd mp4 box in init fragment
for live streams. This ensures qtdemux will sent a segment event prior
to playback start
Test Procedure: Linear channel playback should work as expected
Risks: Low

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

Deepikasri1234 committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

pstroffolino committed:
VPLAY-9889_2 include Content-Length in logging (#176)

* VPLAY-9889_2 include Content-Length in logging

Reason for change: allow visualization to show percent of download completed when timing out

* VPLAY-9889 add contentlength to CurlDownloader

Reason for Change: include total size (when known) in new curl-write logs

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

shripadbpersonal committed:
Merge pull request #194 from rdkcentral/feature/VPLAY-9931_sprint

RDKEMW-5467 : Set PlayerState to IDLE during stop

nu641001 committed:
VPLAY-9999: Progress value for latency displayed as wrong for LLD (#192)

* VPLAY-9999: Progress value for latency displayed as wrong for LLD

Reason for change:Introducing an error log to capture manifest issues
Risks: Low
Test Procedure: Test with channel 922
Priority: P1
Signed-off-by: nu641001 <nu641001@gmail.com>

* VPLAY-9999: Progress value for latency displayed as wrong

Signed-off-by: nu641001 <nu641001@gmail.com>

---------

Signed-off-by: nu641001 <nu641001@gmail.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

jfagunde committed:
VPLAY-10043 Not enable AAMP TSB for FOG urls (#165)

* VPLAY-9842 Not enable AAMP TSB for FOG urls

Reason for Change: Disable AAMP TSB if FOG is used
Test Procedure: Please see the ticket
Risks: Low

Signed-off-by: Jose Fagundez <jfagunde@synamedia.com>

* Add manifest parameter to IsFogUrl()

* Add L1 tests

* Check IsFogUrl only once

* Simplify L1 tests

* L1 test review comment

* Allow a single local TSB and extend L1 tests

* Defog the URL if FOG is not used

* L1 test comments

---------

Signed-off-by: Jose Fagundez <jfagunde@synamedia.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

rajat698 committed:
VPLAY-10062 Build libaampKotlin library for simulator builds only (#198)

Reason for change: To enable kotlin build for simluator only
Risks: Low
Priority: P1

Signed-off-by: rajat698 <emailofrajatyadav@gmail.com>

pstroffolino committed:
VPLAY-10074 FakePlayerSecManager.cpp:170:9: warning: reference to stack memory associated with local variable 'callback' returned [-Wreturn-stack-address] (#201)

Reason for Change: avoid compilation warning with utests
Test Procedure: this warning no longer present when building l1tests
Risks: nonen

shripadgithub committed:
VPLAY-9946: : fixed L1 errors

Reason for change:
Test Procedure:
Priority: P2
Risks:Low

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

srikanthreddybijjam-comcast committed:
VPLAY-9707:Fix multi-character character constant warnings (#205)

Reason for change: : Rectified Multi char Warnings issues in L1.

Test Procedure:  Refer VPLAY-9707 for clearing warnings due to type conversion errors.
Priority: P2

Signed-off-by: srikanthreddybijjam-comcast <srikanthreddybijjam.2000@gmail.com>

Vinish100 committed:
VPLAY-10093: Log flooding with Linear SLD channels

Reason for change: Fix log flooding by changing the log
levels
Test Procedure: Look out for log flooding of mentioned
logs during linear playback
Risks: None

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

Commit:
VPLAY-9946: L1 errors fixed

Reason for change: Added expect call to failing tests
Test Procedure: L1 tests should pass
Priority: P2
Risks:Low

Signed-off-by: asasik397 <Abhijith_Sasikumar@comcast.com>

pstroffolino committed:
VPLAY-9935 warning: initialized lambda captures are a C++14 extension [-Wc++14-extensions] (#210)

Reason for Change: address longstanding compilation warning

Test Procedure: compilation warning no longer seen
Risks: Low

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

pstroffolino committed:
VPLAY-10116 FakePrivateInstance includes implementation that can crash (#215)

* VPLAY-10116 FakePrivateInstance includes implementation that can crash at runtime

Reason for Change: use of deallocated memory crash seen in some l1 tests
Test Procedure: l1 tests 100% building and passing on both ubuntu and Mac
Risks: Low (no core aamp changes)

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

* VPLAY-10116 "FakePrivateInstance simplifications

Reason for Change: removing commented out code, fix spellings
Test Guidance: all l1 tests passing - no crash
Risk: Low

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

---------

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

Deepikasri1234 committed:
Merge branch 'dev_sprint_25_2' into feature/RDK-56050

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

shripadbpersonal committed:
Merge pull request #199 from rdkcentral/feature/VPLAY-9946_sprint

VPLAY-9946: Playback failed in for certain live channels

shripadbpersonal committed:
Merge branch 'dev_sprint_25_2' into feature/VPLAY-10086_sprint

shripadbpersonal committed:
Merge pull request #226 from rdkcentral/feature/VPLAY-10086_sprint

VPLAY-10093: Log flooding with Linear SLD channels

Deepikasri1234 committed:
RDK-56050 Merging of OCDM, gst-plugins-rdk into Library - Design

Reason for change: for dev_Sprint_25_2
Test Procedure: Refer Ticket

Risks: Low

Priority: P1

Change-Id: If0a31d066478806bae29f09ab829b1108d7856ba
Signed-off-by: Deepikasri N <Deepikasri_N@Comcast.Com

Gnanesha committed:
VPLAY-10178 : application getting crashed due to async task thread leak (#232)

* VPLAY-10178 : application getting crashed due to async task thread leak

Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431
Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

* VPLAY-10178 : application getting crashed due to async task thread leak

Reason for change: delete the InterfacePlayerRDK instance upon deleteing the aamp instance
Risks: Low
Test Procedure: Please see the ticket
Priority: P1

Change-Id: Ie08e86e92cce168be1cc2e9188839ba6951dd431
Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

---------

Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>
Co-authored-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

Vinish100 committed:
VPLAY-9947: Fix compilation issues in MAC Simulator (#225)

* VPLAY-9947: Fix compilation issues in MAC Simulator

Reason for change: Enforce C++ version to be used in the
CMakeLists.txt files throughout AAMP to avoid compilation
errors. Bump up CMake required version to 3.5 which is required
in latest MAC OS. Add env variable for building AAMP dependencies
in MAC without patching source files
Test Procedure: Compilation should be successful in all platforms
Risks: Low

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

* VPLAY-9947: Fix compilation issues in MAC Simulator

Reason for change: Refactor CMakeLists.txt. Add option
to downgrade asio dependent library to 1.18.2 in MAC for
building subtec. Fix device build errors and L1.
Test Procedure: Compilation should be successful in all platforms
Risks: Low

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

* VPLAY-9947: Fix compilation issues in MAC Simulator

Reason for change: Fix identation and add license headers.
Fix L2 test failure in Ubuntu.
Test Procedure: Compilation should be successful in all platforms
Risks: Low

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

* VPLAY-9947: Fix compilation issues in MAC Simulator

Reason for change: Skip building glib-2.0 manually and fixed
build issues in Ubuntu 24.04, address review comments.
Test Procedure: Compilation should be successful in all platforms
Risks: Low

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

---------

Signed-off-by: Vinish100 <vinish.balan@gmail.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

Commit:
DISABLE_SECURITY_TOKEN secmanager support

rekhap2kandhavelan committed:
Update L1-tests.yml

VNagori committed:
VPLAY-9864 :  Set available property of audio to false if not present… (#216)

* VPLAY-9864 :  Set available property of audio to false if not present in AAMP TSB

* Update the AAMP-UVE-API doc

---------

Co-authored-by: shripadbpersonal <105402691+shripadbpersonal@users.noreply.github.com>

anjali-syna committed:
VPLAY-10085 - AV freeze in live-pause-seekToLive scenario (#209)

* VPLAY-10085 [ENT-4082] AV freeze in live-pause-seekToLive scenario

Reason for change: AV freeze is observed in L2 tests while performing
  seek to live after live-pause.

Changes:
* Setting 'mLocalAAMPInjectionEnabled' flag to true when TuneHelper is
* called with tuneType 'eTUNETYPE_SEEKTOLIVE' and there is data
* available in the TSB. This enables AampTsbReader to read and feed
* the available data in TSB to gstreamer

Test Procedure: Run L2 tests 5001 and 5002

* VPLAY-10085 [ENT-4082] AV freeze in live-pause-seek to live scenario

Reason for change: AV freeze is observed in L2 tests while performing
  seek to live after live-pause.

Changes:
* Setting 'mLocalAAMPInjectionEnabled' flag to true when TuneHelper is
* called with tuneType 'eTUNETYPE_SEEKTOLIVE' and there is data
* available in the TSB. This enables AampTsbReader to read and feed
* the available data in TSB to gstreamer

Test Procedure: Run L2 tests 5001 and 5002

Risks: Low

Signed-off-by: anjali-syna <206662904+anjali-syna@users.noreply.github.com>

* VPLAY-10085 [ENT-4082] AV freeze in live-pause-seek to live scenario

Reason for change: AV freeze is observed in L2 tests while performing
  seek to live after live-pause.

Changes:
* Setting 'mLocalAAMPInjectionEnabled' flag to true when TuneHelper is
* called with tuneType 'eTUNETYPE_SEEKTOLIVE' and there is data
* available in the TSB. This enables AampTsbReader to read and feed
* the available data in TSB to gstreamer

Test Procedure: Run L2 tests 5001 and 5002

Risks: Low

Signed-off-by: anjali-syna <206662904+anjali-syna@users.noreply.github.com>

* VPLAY-10085 [ENT-4082] AV freeze in live-pause-seek to live scenario

Reason for change: AV freeze is observed in L2 tests while performing
  seek to live after live-pause.

Changes:
* Setting 'mLocalAAMPInjectionEnabled' flag to true when TuneHelper is
* called with tuneType 'eTUNETYPE_SEEKTOLIVE' and there is data
* available in the TSB. This enables AampTsbReader to read and feed
* the available data in TSB to gstreamer

Test Procedure: Run L2 tests 5001 and 5002

Risks: Low

Signed-off-by: anjali-syna <206662904+anjali-syna@users.noreply.github.com>

---------

Signed-off-by: anjali-syna <206662904+anjali-syna@users.noreply.github.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

DomSyna committed:
VPLAY-9825 [ENT-4082] Wrong fragment cache being checked (#166)

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

In various places in the code the wrong fragment cache is being checked
when local TSB is enabled

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

In various places in the code the wrong fragment cache is being checked
when local TSB is enabled

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

In various places in the code the wrong fragment cache is being checked
when local TSB is enabled

Fixed crash due due to memset

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

* VPLAY-9825 [ENT-4082] Wrong fragment cache being checked

When the initial change to use the single fragment cache was done, some
code was missed.

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>

---------

Signed-off-by: DomSyna <192202460+DomSyna@users.noreply.github.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

shripadgithub committed:
Revert "VPLAY-9525: Switching between tune locators not working without explicit stop"

This reverts commit 87c0700957ce70f67fdd194336c80abc06421e58.

shripadgithub committed:
Revert "RDKEMW-5467 : Set PlayerState to IDLE during stop"

This reverts commit 1ee5e66bdaaf8696e5d7533ea4b741164c93455e.

shripadgithub committed:
Revert "VPLAY-9063: AAMP Instance: Send state changed event only after full stop"

This reverts commit ef6208855ddf0d5b25f45ce61c0a8c939fb3d3a9.

Vinish100 committed:
VPLAY-10204: RDK-E Build issues with Federated 2507 (#250)

Reason for change: Fix RDK-E build issues seen with
federated 2507 tag
Test Procedure: Make sure RDK-E build passes
Risks: None

Signed-off-by: Vinish100 <vinish.balan@gmail.com>

rajat698 committed:
VPLAY-10058  AAMP logging fix - Sending Time Timemetric -> Sending TuneTimeMetric event (#197)

Reason for change: Tracking suggestion to change logging from odd spelling/phrasing "Time Timemetric"
Risks: Low
Priority: P1

Signed-off-by: rajat698 <emailofrajatyadav@gmail.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

VNagori committed:
VPLAY-10118 : Set available property of text to true if present in AAMP TSB (#239)

Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

anshephe committed:
VPLAY-9868 [ENT-4082] AAMP TSB with CDAI - Handle Advert At Live Point (#234)

VPLAY-9868 [ENT-4082] AAMP TSB with CDAI - Handle Advert At Live Point

Reason for Change: Incorrect pts offset being used in GetFirstPts when playback is from Aamp TSB
* Corrected updating of mFirstPTS
* Added/updated utests
* Limited change to just fix the incorrect PTS offset used in GetFirstPTS
* Fixed whitespace
* Undo change to InitTsbReaderTest
* Resolved review comments.

Risk: Low

Signed-off-by: Andrew Shepherd <anshephe@synamedia.com>

rkandh015 committed:
Rebase with dev_sprint_25_2

ALSAMEEMA committed:
RDK-56194: Integrating ContentProtection Firebolt in AAMP (#247)

Reason for change: Interim change to avoid playback failure and allow DTM-4118 changes to be merged independently
		   as it replaces shared memory message passing of license metadata to string format.
		   Will be removed once RDK-56194 landed on sprint

Test Procedure: Encrypted Playback should pass
Risks: Low
Priority: P1

Signed-off-by: Alsameema <alsameema_ahmedansar@comcast.com>
Co-authored-by: Alsameema <alsameema_ahmedansar@comcast.com>
Co-authored-by: pstroffolino <Philip_Stroffolino@cable.comcast.com>

rkandh015 committed:
rebase with dev_sprint

pstroffolino committed:
VPLAY-9063 l1 fixes to go along with revert

Reason for Change: revert to avoid blocker; see ticket

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

rekhap2kandhavelan committed:
Update L1-tests.yml

rekhap2kandhavelan committed:
Update L1-tests.yml

nebutler963 committed:
VPLAY-10194 -AAMP Tune() leaking mCdaiObject Object (#249)

VPLAY-10194 -AAMP Tune() leaking mCdaiObject Object

Reason for Change: When tuning mCdaiObject is leaked.
Test Procedure: L1/L2 Full Stack testing
Risks: Low

jameslofthouse-RED committed:
VPLAY-9355 remove catch-up delay when processing a discontinuity (#212)

* VPLAY-9355 Underflow causes aamp to go into pause state
Reason for change: If audio is processing a discontinuity, do not wait for it to catch up to video
Test Procedure: See ticket
Risks: Low

Signed-off-by: James Lofthouse <james_lofthouse@comcast.com>

pstroffolino committed:
rdk-56194 typo fix (#261)

RDK-56194: Integrating ContentProtection Firebolt in AAMP

Reason for Change: spelling fix

Signed-off-by: Stroffolino <philip_stroffolino@cable.comcast.com>

pstroffolino committed:
Merge branch 'dev_sprint_25_2' into feature/feature/VPLAY-9063_revert_sprint

pstroffolino committed:
Merge pull request #257 from rdkcentral/feature/feature/VPLAY-9063_revert_sprint

VPLAY-10224 : revert of VPLAY-9525 , RDKEMW-5467 VPLAY-9063

Commit:
MonitorAVState avposition data type change

srikanthreddybijjam-comcast committed:
VPLAY-9989: Fix for AUTO_CAUSES_COPY Coverity Issue  (#206)

VPLAY-9989:Fix for AUTO_CAUSES_COPY Coverity Issue

Reason for change: Resolved Coverity Issue caused by(AUTO_CAUSES_COPY)
Test Procedure:Refer VPLAY-9989 Coverity Issue(AUTO_CAUSES_COPY).
Priority: P1

Signed-off-by: srikanthreddybijjam-comcast<srikanthreddybijjam.2000@gmail.com>

varatharajan568 committed:
RDK-58039 Move the aamp-xternal-stats header file to the Middleware directory (#188)

Reason for Change: Moved & renamed the aamp-xternal-stats.h as
player-xternal-stats.h due to it is an Middleware specific file
Test Procedure: None
Risks: Low

Signed-off-by: Varatharajan_Narayanan <Varatharajan_Narayanan@comcast.com>

psiva01 committed:
VPLAY-10196: Observed Black screen with No AV on playing ColdDVR asset (#262)

VPLAY-10196: Reason for change: Clear duration value in mdhd mp4 box in init fragment
for DVR streams as well. This ensures qtdemux will sent a segment event prior
to playback start
Test Procedure: DVR playback should work
Risks: Low

Signed-off-by: Sivasubramanian <sivasubramanian.patchaiperumal@ltts.com>

rkandh015 committed:
rebase with dev_sprint

Change-Id: I77d0d29b44a01ae8d9fb3ddd455815700a3dbc76
